index.html

```diff
@@ -307,5 +308,6 @@ <h1 class="text-2xl font-semibold tracking-[7px] text-blue-0">TODO</h1>
     </footer>
     <script type="module" src="src/js/main.js"></script>
     <script type="module" src="src/js/auth.js"></script>
+    <script type="module" src="src/js/todoFunctionality.js"></script>
```

Comment:
These should be removed and let's use main.js as the only main js entry code. 

All js code should go through the main.js.